src/java.base/share/classes/java/security/KeyPairGenerator.java

```diff
@@ -68,16 +68,10 @@
  * <p>Since no other parameters are specified when you call the above
  * algorithm-independent {@code initialize} methods, it is up to the
  * provider what to do about the algorithm-specific parameters (if any) to be
- * associated with each of the keys.
- *
- * <p>If the algorithm is the <i>DSA</i> algorithm, and the keysize (modulus
- * size) is 512, 768, 1024, or 2048, then the <i>Sun</i> provider uses a set of
- * precomputed values for the {@code p}, {@code q}, and
- * {@code g} parameters. If the modulus size is not one of the above
- * values, the <i>Sun</i> provider creates a new set of parameters. Other
- * providers might have precomputed parameter sets for more than just the
- * modulus sizes mentioned above. Still others might not have a list of
- * precomputed parameters at all and instead always create new parameter sets.
+ * associated with each of the keys. Information about how the <i>Sun</i> prover
+ * handles such algorithm-specific parameters can be found in the
+ * <a href="https://docs.oracle.com/en/java/javase/11/security/oracle-providers.html">
```

Comment:
You should use a link like "{@extLink security_guide_jdk_providers JDK Providers}" so that it links to the version of the guide that is the same as the JDK version. See [this](https://github.com/openjdk/jdk/blob/9768f60a945db277ded5e37ecba9ec34421d9968/src/java.base/share/classes/java/security/AlgorithmParameterGeneratorSpi.java#L43) for an example. 

Also, I would make this text more generic and not specific to the Sun provider. For example:

"See the Keysize Restrictions sections of the {@extLink security_guide_jdk_providers JDK Providers}
document for information on the default keysizes used by JDK providers."